page-optimize.php

```diff
@@ -100,7 +102,8 @@ function page_optimize_should_concat_js() {
 		return $_GET['concat-js'] !== '0';
 	}
 
-	return !! get_option( 'page_optimize-js', page_optimize_js_default() );
+	$should_concat = !! get_option( 'page_optimize-js', page_optimize_js_default() );
+	return apply_filters( 'page_optimize_should_concat_js', $should_concat );
```

Comment:
While we're revisiting conditional loading, we might as well support a first-class should-concat filter for JS.